### PR TITLE
Draft: Implement pyze functionnality (v2)

### PR DESCRIPTION
**Pull request by epenet**

### CODE REVIEW

**Comment by epenet** [comment]
This PR is too big => converted to draft

File: src/renault_api/renault_client.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from __future__ import annotations
+
+import logging
+from typing import Any
+from typing import List
+
+from aiohttp import ClientSession
+
+from . import gigya
+from . import renault_account
```
Comment (Quentame) [inline src/renault_api/renault_client.py:11]:
```suggestion
from .renault_account import RenaultAccount
```

File: src/renault_api/renault_client.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
+"""Client for Renault API."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+from typing import List
+
+from aiohttp import ClientSession
+
+from . import gigya
```
Comment (Quentame) [inline src/renault_api/renault_client.py:10]:
```suggestion
from .gigya import Gigya
```

**Review by Quentame — CHANGES_REQUESTED** [review]
Some typing, otherwise LGTM

**Pull request closed without merging**